browser/ui/webui/settings/brave_sync_handler.cc

```diff
@@ -83,12 +84,11 @@ void BraveSyncHandler::HandleGetSyncCode(const base::ListValue* args) {
   const base::Value* callback_id;
   CHECK(args->Get(0, &callback_id));
 
-  brave_sync::Prefs brave_sync_prefs(profile_->GetPrefs());
-  std::string sync_code = brave_sync_prefs.GetSeed();
-  if (sync_code.empty()) {
-    std::vector<uint8_t> seed = brave_sync::crypto::GetSeed();
-    sync_code = brave_sync::crypto::PassphraseFromBytes32(seed);
-  }
+  auto* sync_service =
+    static_cast<syncer::BraveProfileSyncService*>(GetSyncService());
```

Comment:
lint nit: 4 spaces for continuation of a line

Reply:
also see https://github.com/brave/brave-core/pull/6565/files#r482274558

Reply:
addressed in https://github.com/brave/brave-core/pull/6565/commits/3ce15e18d2629cc8017b6a39459249389f0d7cd8